packages/fxa-auth-server/lib/senders/emails/templates/newDeviceLogin/includes.ts

```diff
@@ -0,0 +1,28 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { GlobalTemplateValues } from '../../../renderer';
+
+export const getIncludes = (clientName: string): GlobalTemplateValues => {
+  const subject =
+    clientName === 'Mozilla account'
```

Comment:
Should this be 'Mozilla' because of:
https://github.com/mozilla/fxa/blob/fbf452be0538b9c9d6216ad7b50205323dda9018/packages/fxa-auth-server/lib/senders/email.js#L816

I was testing locally and the subject line was still coming through as "New sign-in to Mozilla"

Reply:
After changing line 9 above to 'Mozilla', I do see the subject line line changed to "New sign-in to your Mozilla account" (see below in maildev):
![image](https://github.com/mozilla/fxa/assets/22231637/fd9b832c-5253-4382-a2bd-10b5ce9ba7d1)